reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md

```diff
@@ -155,15 +166,16 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
    PubkeyAuthentication yes
    ```
 
-1. Restart the **sshd** service.
+1. Restart the **ssh** service.
 
    ```bash
-   sudo service sshd restart
+   sudo service ssh restart
    ```
 
 ## Set up on a macOS computer
 
-1. Install the latest version of PowerShell, see [Installing PowerShell Core on macOS](../../install/installing-powershell-core-on-macos.md).
+1. Install the latest version of PowerShell, see
```

Comment:
This reads like two sentences. I'd split it up or reword the first half.

Reply:
Fixed